packages/web-app-external/src/components/ErrorScreen.vue

```diff
@@ -1,6 +1,23 @@
 <template>
   <div class="uk-text-center">
     <oc-icon size="xxlarge" name="warning" />
-    <p v-translate>Error when loading the application</p>
+    <h1 v-translate class="oc-text-lead">Error when loading the application</h1>
+    <p v-if="message" v-translate>{{ message }}</p>
```

Comment:
No need for the v-translate, we can't know what the message is in Transifex (the translation tool)